src/js/nav/globalNav.js

```diff
@@ -10,24 +13,24 @@ export let getNavFromConfig = async (masterConfig) => {
     }, {});
 };
 
+const isCurrVisible = (permissions) => Promise.all(
+    flatMap([permissions], ({ method } = {}) => (
+        // (null, undefined, true) !== false
+        visibilityFunctions?.[method]?.(...permissions.args || []) !== false || false
```

Comment:
I don't think the `|| false` is necessary here. x !== `false` will always return boolean value. Or am I crazy :laughing:  I must say this makes me think much harder than it should.

Reply:
Yeah, the false is not necessary here, the logic is `if visibility function return something else then false return true, otherwise false`. But that's correct and doesn't need the `|| false`